src/main/webapp/resources/js/components/samples/components/fastqc/FastQC.tsx

```diff
@@ -47,6 +47,10 @@ export function FastQC() {
     );
   }, [sequencingObjectId, fileId]);
 
+  const textLevel = {
```

Comment:
Because of a typescript error:

`TS2322: Type '{ children: any; level: number; strong: true; className: string; }' is not assignable to type 'IntrinsicAttributes & TextProps & { children?: ReactNode; }'.   Property 'level' does not exist on type 'IntrinsicAttributes & TextProps & { children?: ReactNode; }'.`

Found that creating a constant for it made the error go away :)

Reply:
You are getting that error because as it says `Property 'level' does not exist on type 'IntrinsicAttributes & TextProps & { children?: ReactNode; }`.  I think instead of `Typography.Text`, you are thinking of `Typography.Title` which does take a `level` attribute.

Reply:
Removed in 445c58cec70fbfbbe6113426d30eb0caf860e90f